src/drawer.js

```diff
@@ -162,7 +162,7 @@ WaveSurfer.Drawer = {
                 this.recenterOnPosition(newPos);
             }
 
-            this.updateProgress(progress);
+            this.updateProgress(progress, true);
```

Comment:
The `updateProgress` method takes only one argument. https://github.com/katspaugh/wavesurfer.js/blob/master/src/drawer.canvas.js#L103